### PR TITLE
add pricing breakdown page

### PR DESCRIPTION
**Pull request by drakirnosslin**
## Affected Components
* [ ] Content & Marketing
* [ ] Pricing
* [ ] Test
* [ x ] Docs
* [ ] Learn
* [ ] Other

## Pre-Requisites
* [ ] Code is linted (`npm run lint`)

### CODE REVIEW

**Review by maxigimenez — DISMISSED** [review]

File: site/content/docs/monitoring/check-pricing.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+title: Check Pricing
```
Comment (ragog) [inline site/content/docs/monitoring/check-pricing.md:2]: should be "Check pricing"

File: site/content/docs/monitoring/check-pricing.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+weight: 9
+menu:
+   platform:
+    parent: "Monitoring"
+---
+
+Check pricing is based on a number of factors. Here is a breakdown of the things that determine the cost of a check.
```
Comment (ragog) [inline site/content/docs/monitoring/check-pricing.md:9]: @drakirnosslin would a breakdown summary of check types at the beginning here help people who might be new to Checkly quickly understand what is and is not relevant to their case?

File: site/content/docs/monitoring/check-pricing.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+Balance the number of locations the check is using to ensure you are quickly made aware of potential issues in critical locations without running an unnecessary amount of checks.
+
+Continuing our example from earlier, if we want our Browser check to run in parallel from 5 locations, the monthly cost will increase from $3,65 to $18,25.
+
+### Retries
+
+When a check is [retried](/docs/alerting-and-retries/retries), this counts as a new check run. A flaky check can increase your check run costs above the expected. If you have problems with flaky checks, ask our community or our support for tips on how to improve check stability and reduce cost.
+
+If our check from earlier has a 20% retry rate, this will increase our cost from $3,65 to $4,40.
```
Comment (ragog) [inline site/content/docs/monitoring/check-pricing.md:31]: I would change this from "our cost" to "the cost" or "your cost" to make it as unambiguous as possible

File: site/content/docs/monitoring/check-pricing.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+title: Check Pricing
+weight: 9
+menu:
+   platform:
+    parent: "Monitoring"
+---
+
+Check pricing is based on a number of factors. Here is a breakdown of the things that determine the cost of a check.
+
+### Base cost
```
Comment (ragog) [inline site/content/docs/monitoring/check-pricing.md:11]: These should be # or ## maybe?

**Review by ragog — CHANGES_REQUESTED** [review]
@drakirnosslin some nits. Feels like the page might be a bit "hostile" for people who have found Checkly recently and need to understand our pricing quickly.

File: site/content/docs/monitoring/check-pricing.md (hunk 2 above)
Comment (tnolet) [reply site/content/docs/monitoring/check-pricing.md:9]: no, let's keep the page focused. It needs to be short and scannable.

**Review by tnolet — COMMENTED** [review]
It's a bit hard to scan. I think the page needs to look like a "pricing table deluxe" only focussing on the breakdown of pricing.

Maybe just use an elaborate markdown table? I use this to do that https://www.tablesgenerator.com/markdown_tables

File: site/content/docs/monitoring/check-pricing.md (hunk 2 above)
Comment (ragog) [reply site/content/docs/monitoring/check-pricing.md:9]: @tnolet you're probably right. Would you structure it like this?
Screenshot 2024-01-03 at 15 12 29

**Comment by modern-sapien** [comment]
Maybe we could add these to a section before we get into the math for folks to have an easier time understanding checks and then they can dive into the docs if they want to follow along?

| Single Step API | Parallel Locations | # of Retries | Credits Used |
|-----------------|---------------------|--------------|--------------|
| 1               | 2                   | 0            | 2            |
| 1               | 2                   | 2            | 4            |


| Multi Step | Number of Steps | Parallel Locations | # of Retries | Credits Used |
|------------|-----------------|---------------------|--------------|--------------|
| 1          | 3               | 2                   | 0            | 6            |
| 1          | 3               | 2                   | 2            | 4            |


| Browser Check | Parallel Locations | # of Retries | Credits Used |
|---------------|---------------------|--------------|--------------|
| 23            | 3                   | 2            | 138          |
| 23            | 3                   | 2            | 276          |


or one table for all of them

| Type           | Number of Steps | Parallel Locations | # of Retries | Credits Used |
|----------------|-----------------|---------------------|--------------|--------------|
| Single Step API| N/A             | 2                   | 0            | 2            |
| Single Step API| N/A             | 2                   | 2            | 4            |
| Multi Step     | 3               | 2                   | 0            | 6            |
| Multi Step     | 3               | 2                   | 2            | 4            |
| Browser Check  | N/A             | 3                   | 2            | 138          |
| Browser Check  | N/A             | 3                   | 2            | 276          |

@tnolet @ragog @drakirnosslin

**Comment by ragog** [comment]
> Maybe we could add these to a section before we get into the math for folks to have an easier time understanding checks and then they can dive into the docs if they want to follow along?
> 
> Single Step API	Parallel Locations	# of Retries	Credits Used
> 1	2	0	2
> 1	2	2	4
> Multi Step	Number of Steps	Parallel Locations	# of Retries	Credits Used
> 1	3	2	0	6
> 1	3	2	2	12
> Browser Check	Parallel Locations	# of Retries	Credits Used
> 23	3	2	138
> 23	3	2	276
> or one table for all of them
> 
> Type	Number of Steps	Parallel Locations	# of Retries	Credits Used
> Single Step API	N/A	2	0	2
> Single Step API	N/A	2	2	4
> Multi Step	3	2	0	6
> Multi Step	3	2	2	12
> Browser Check	N/A	3	2	138
> Browser Check	N/A	3	2	276
> @tnolet @ragog @drakirnosslin

@modern-sapien not sure I'd talk credits here, I'd rather keep it in the language of the Pricing page.

**Comment by tnolet** [comment]
> Maybe we could add these to a section before we get into the math for folks to have an easier time understanding checks and then they can dive into the docs if they want to follow along?
> 
> Single Step API	Parallel Locations	# of Retries	Credits Used
> 1	2	0	2
> 1	2	2	4
> Multi Step	Number of Steps	Parallel Locations	# of Retries	Credits Used
> 1	3	2	0	6
> 1	3	2	2	12
> Browser Check	Parallel Locations	# of Retries	Credits Used
> 23	3	2	138
> 23	3	2	276
> or one table for all of them
> 
> Type	Number of Steps	Parallel Locations	# of Retries	Credits Used
> Single Step API	N/A	2	0	2
> Single Step API	N/A	2	2	4
> Multi Step	3	2	0	6
> Multi Step	3	2	2	12
> Browser Check	N/A	3	2	138
> Browser Check	N/A	3	2	276
> @tnolet @ragog @drakirnosslin

No, this is too complex. I want one, scannable table with some voice over. Having two entries for the same check type is not useful.

File: site/content/docs/monitoring/check-pricing.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+## Overview
+
+Checkly supports different kinds of checks which are billed slightly differently:
+
+| Check type | Base cost          | Retries             | Parallel scheduling    |
+|------------|--------------------|---------------------|------------------------|
+| Browser    | Number of runs     | Each retry is a run | Each location is a run |
+| API        | Number of runs     | Each retry is a run | Each location is a run |
+| Multistep  | Number of requests | Retries x requests  | Locations x runs       |
```
Comment (drakirnosslin) [inline site/content/docs/monitoring/check-pricing.md:17]: Should this be 'Locations x requests' ?

Comment (ragog) [reply site/content/docs/monitoring/check-pricing.md]: @drakirnosslin yep! good catch

**Comment by tnolet** [comment]
@ragog it's just missing the most obvious part in the table to make it scannable. The $$ / unit.

CleanShot 2024-01-03 at 18 02 50

**Review by modern-sapien — DISMISSED** [review]

**Review by tnolet — APPROVED** [review]
LGTM

**Pull request merged**